sample/android/build.gradle

```diff
@@ -90,7 +90,7 @@ dependencies {
     implementation libraries.okhttp
     implementation libraries.moshiKotlin
     implementation libraries.moshi
-    implementation("io.ktor:ktor-client-okhttp:1.3.2")
+    implementation("io.ktor:ktor-client-okhttp:1.3.1")
```

Comment:
Dependency couldn't be found with 1.3.2. I aligned it with the other ktor versions before which work